File: java/src/main/java/com/ellen/java/jmvp/di/component/LoginComponent.java
package com.ellen.java.jmvp.di.component;

import com.ellen.java.jmvp.di.module.LoginModule;
import com.ellen.java.jmvp.login.LoginActivity;

import dagger.Component;

@Component(modules = {LoginModule.class})
public interface LoginComponent {
    void injectLoginActivity(LoginActivity loginActivity);
}

File: app/src/main/java/com/ellen/androidarchitecture/di/component/LoginComponent.kt
package com.ellen.androidarchitecture.di.component

import com.ellen.androidarchitecture.di.module.LoginModule
import com.ellen.androidarchitecture.kmvp.login.LoginActivity
import com.ellen.androidarchitecture.kmvp.login.LoginPresenter
import dagger.Component

@Component(modules = [LoginModule::class])
interface LoginComponent {
    fun injectLoginActivity(loginActivity: LoginActivity)
}